spec/support/mockers/equipment_model.rb

```diff
@@ -0,0 +1,31 @@
+require Rails.root.join('spec/support/mockers/mocker.rb')
```

Comment:
I wasn't able to figure out why, but it doesn't seem like the files in `spec/support` get automatic access to everything in `spec_support`. It might be because `spec_helper` isn't included here, but I don't think it's appropriate to add it in.